microsoft/testsuites/lis/lissuite.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.tools import Wget
+from lisa.util import SkippedException
+
+
+@TestSuiteMetadata(
+    area="lis",
+    category="functional",
+    description="""
+    ##################################
+    """,
+)
+class lis(TestSuite):  # noqa
+    @TestCaseMetadata(
+        description="""
+            #################
+        """,
+        priority=1,
+    )
+    def verify_lis_driver_version(self, node: Node) -> None:
+        wget_tool = node.tools[Wget]
+        rpm_status = node.execute(
+            'rpm -qa | grep "kmod-microsoft-hyper-v" && rpm -qa '
+            '| grep "microsoft-hyper-v"'
+        ).exit_code
+
+        if rpm_status != 0:
+            raise SkippedException("No LIS RPM's are detected. Skipping test.")
+        version = node.execute(
+            """modinfo "hv_vmbus" | grep version: | head -1 | awk '{print $2}'"""
+        ).stdout
+
+        for i in range(5, 8):
```

Comment:
Add comments to explain the purpose, it's not easy to get.

Reply:
It looks downloaded some header files, and check if it's consistent with system. What's the logic is validating?

Reply:
It downloads header files based on the LIS version and compares the installed LIS version to the expected one which is found in the header files.

Reply:
ok, add comments in code to explain the logic.